source/includes/example-connect.rst

```diff
@@ -50,3 +50,24 @@ following command on one line:
                        clusterTwo02.fancyCorp.com:20020,
                        clusterTwo03.fancyCorp.com:20020
 
+To use ``mongosync`` with Atlas clusters, you add the
+:urioption:`ssl=true <ssl>` option to enable SSL. For example:
```

Comment:
One small tweak -- I think it would be a little more clear if this read something like:
`Atlas clusters require TLS/SSL connections, so to use mongosync with Atlas clusters, you add the `ssl=true` option.

Reply:
Done.